org.eclipse.scanning.api/src/org/eclipse/scanning/api/AbstractScannable.java

```diff
@@ -259,6 +263,7 @@ public MonitorRole getMonitorRole() {
 		return monitorRole;
 	}
 	public MonitorRole setMonitorRole(MonitorRole monitorRole) {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1161)